doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md

```diff
@@ -2,7 +2,12 @@
 Content:
 
 * [Functionality](#section1)
```

Comment:
Not sure if this is the right .md to do the documentation changes.

ngsi_postgis_sink.md is basically a pointer to ngsi_postgresql_sink.md. Maybe the changes should be done in ngsi_postgresql_sink.md? Or they are Postgis-specific and the new funciontaltiy wouldn't work with plain Postgresql sink?

Reply:
That's correct, this changes only applies to Postgis sink... Right @manucarrace?

Reply:
Discussed at skype. Same functionality for Postgresql will come in another PR.

NTC